### PR TITLE
Build site extension during main build

### PR DESCRIPTION
**Pull request by pakrym**
And remove the runtime site extension as it was moved to AspNetCore

### CODE REVIEW

File: src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj [hunk 1]
```diff
@@ -10,12 +10,24 @@
...
   </PropertyGroup>
 
   <ItemGroup>
-    <HostingStartupPackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(MicrosoftAspNetCoreAzureAppServicesHostingStartupPackageVersion)" />
+    <HostingStartupPackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(PackageVersion)" />
 
+    <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
     <PackageReference Include="Internal.AspNetCore.SiteExtension.Sdk" Version="$(InternalAspNetCoreSiteExtensionSdkPackageVersion)" PrivateAssets="All" />
+
+    <ExcludePackageFileFromSigning  Include="Microsoft.Extensions.Logging.AzureAppServices.dll" />
```
Comment (natemcmaster) [inline src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: Uh, why? Shouldn't we sign everything?

Comment (pakrym) [reply src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: `Microsoft.Extensions.Logging.AzureAppServices` is a package reference so it should be already signed.

Comment (natemcmaster) [reply src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: Ok.

**Comment by natemcmaster** [comment]
The CI build isn't happy :-/

**Pull request closed without merging**